### PR TITLE
feat: add listApps and activeAppInfo

### PR DESCRIPTION
**Pull request by KazuCocoa**
@jlipps I have added two commands to get installed app information. They should help to check especially installed apps. They cannot be automate via chrome but maybe can via socket communication.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,18 +74,20 @@ list):
 |webOS Command|Parameters|Description|
 |-------------|----------|-----------|
 |`pressKey`|`key`, `duration`|Press a remote key for `duration` milliseconds (defaults to 100). The value of `key` must be one of the values listed below|
+|`listApps`| |Return the list of installed applications. `id` key in each value is `appium:appId`. |
```
Comment (jlipps) [inline README.md:77]:
```suggestion
|`listApps`| |Return the list of installed applications. The `id` key in each value is `appium:appId`. |
```

File: lib/cli/ares.js [hunk 2]
```diff
@@ -160,3 +160,31 @@ export async function uninstallApp(appId, deviceName) {
...
+
+
+/**
+ * Return the list of running apps
+ * @param {*} deviceName
+ */
+export async function listRunningApps(deviceName) {
+  //
+  //
+  //com.suitest.watchme.app
```
Comment (jlipps) [inline lib/cli/ares.js:172]: I don't think we need these comments do we?

File: README.md [hunk 3]
```diff
@@ -117,6 +119,59 @@ Here are the accepted values, based on the `appium:rcMode`. Casing does not matt
...
+```
+
+#### webos: activeAppInfo
+
+Response example:
+
+```ruby
+# Ruby
+driver.execute_script "webos: activeAppInfo"
+#=> {"returnValue"=>true, "appId"=>"com.suitest.watchme.app", "processId"=>"", "windowId"=>""}
```
Comment (jlipps) [inline README.md:172]: can we use a different app id example than a suitest one throughout this PR?

**Review by jlipps — CHANGES_REQUESTED** [review]

File: lib/cli/ares.js (hunk 2 above)
Comment (KazuCocoa) [reply lib/cli/ares.js:172]: ooh, these changes are not related. my local past some changes...

**Review by jlipps — APPROVED** [review]

**Pull request merged**